internal/gatewayapi/runner/runner.go

```diff
@@ -231,6 +233,21 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 	r.Logger.Info("shutting down")
 }
 
+func isEndpointRoutingDisabled(resources *gatewayapi.Resources) bool {
+	endpointRoutingDisabled := true
+	if resources.EnvoyProxy != nil {
+		switch ptr.Deref(resources.EnvoyProxy.Spec.RoutingType, v1alpha1.ServiceRoutingType) {
```

Comment:
this should default to `v1alpha1.EndpointRoutingType`

Reply:
ok I guess I got it all backwards